src/SWP/Bundle/RevisionBundle/RevisionContext.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/*
+ * This file is part of the Superdesk Publisher Revision Bundle.
+ *
+ * Copyright 2017 Sourcefabric z.u. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2015 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Bundle\RevisionBundle;
+
+use SWP\Component\Revision\Model\RevisionInterface;
+use SWP\Component\Revision\RevisionContextInterface;
+
+/**
+ * Class RevisionContext.
+ */
+class RevisionContext implements RevisionContextInterface
```

Comment:
this class, as well with models and some interfaces should go to `Revision` component.